### PR TITLE
Update new cohort checklist

### PR DESCRIPTION
**Pull request by yvonne-liu**
Includes specific tasks between CFs and programme directors, relates to #472

### CODE REVIEW

File: curriculum-planning/new-cohort-checklist.md [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
-- [ ] Introduce current mentors to the [gitter](https://gitter.im/) chatroom for `current-mentors` under the `foundersandcoders` organisation.
-- [ ] Send contact email (via [mailchimp](https://mailchimp.com/)) out to FAC alumni to begin arranging external speakers dates for the new curriculum.
-- [ ] Use the Google spreadsheet of past alumni who gave talks / external speakers, stored in `school-curriculum`, to continue arranging external speaker dates for the new curriculum. Update the list from last curriculum if necessary.
+### Prior to the course
+Programme director (supported by FCI if necessary):
+- [ ] Send acceptance email to successful candidates, including a link to the pre-course material
+
+Course Facilitator:
+- [ ] Introduce current mentors to the gitter chatroom for current-mentors under the foundersandcoders organisation.
+- [ ] Create a new github organisation for FAC-X with the following structure. Example.
```
Comment (eliascodes) [inline curriculum-planning/new-cohort-checklist.md]: No link?

File: curriculum-planning/new-cohort-checklist.md [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
+- [ ] Create a gitter chatroom for FAC-X within the foundersandcoders organisation, and delete the "incoming"/"lobby" room.
+- [ ] Add new cohort to Founders & Coders organisation.
+- [ ] Add new cohort to a FAC-X team within this organisation.
+- [ ] Add new cohort to `foundersandcoders` main Gitter room
+- [ ] Add new cohort to `city`-specific Gitter room
+- [ ] Invite the new cohort in for an installation party in the last week of FACX-1 to meet each other and mentors.
+- [ ] Contacting alumni to create a pastoral care team
+
+Weekly curriculum task group along with current CF / maintainers of the `master-reference`:
+- [ ] Ensure all curriculum related files are updated: those in the coursebook general folder & the curriculum (London, Nazareth) repo.
```
Comment (eliascodes) [inline curriculum-planning/new-cohort-checklist.md]: there are no curriculum repos anymore

File: curriculum-planning/new-cohort-checklist.md [hunk 3]
```diff
@@ -1,20 +1,30 @@
...
-- [ ] Send acceptance email to successful candidates, including a link to the precourse material
-- [ ] Introduce current mentors to the [gitter](https://gitter.im/) chatroom for `current-mentors` under the `foundersandcoders` organisation.
-- [ ] Send contact email (via [mailchimp](https://mailchimp.com/)) out to FAC alumni to begin arranging external speakers dates for the new curriculum.
-- [ ] Use the Google spreadsheet of past alumni who gave talks / external speakers, stored in `school-curriculum`, to continue arranging external speaker dates for the new curriculum. Update the list from last curriculum if necessary.
+### Prior to the course
+Programme director (supported by FCI if necessary):
+- [ ] Send acceptance email to successful candidates, including a link to the pre-course material
+
+Course Facilitator:
+- [ ] Introduce current mentors to the gitter chatroom for current-mentors under the foundersandcoders organisation.
```
Comment (astroash) [inline curriculum-planning/new-cohort-checklist.md]: Is there a 'current-mentors' gitter channel? Or is this implying that one needs to be created and them to be added to it? I've currently made individual week chat channels to check in but not sure if this is the 'standard'

File: curriculum-planning/new-cohort-checklist.md (hunk 1 above)
Comment (astroash) [reply curriculum-planning/new-cohort-checklist.md]: I know Londons organisations have been created upto 20 (not sure if this is the same for Gaza/ Nazareth). Is it worth mentioning this here?

Comment (yvonne-liu) [reply curriculum-planning/new-cohort-checklist.md]: I think `current-mentor` is no longer an active Gitter room.

File: curriculum-planning/new-cohort-checklist.md (hunk 3 above)
Comment (yvonne-liu) [reply curriculum-planning/new-cohort-checklist.md]: Oh sorry, responded to that above. I believe `current-mentor` is outdated.

File: curriculum-planning/new-cohort-checklist.md (hunk 1 above)
Comment (eliascodes) [reply curriculum-planning/new-cohort-checklist.md]: @yvonne-liu I think we should say that we (@sofer) owns the fac-13 -> fac-21 organisations

**Comment by eliascodes** [comment]
@yvonne-liu could you also address #516 in this, while you're at it?

**Pull request merged**